mk-files/release.mk

```diff
@@ -90,7 +90,10 @@ gorelease:
 	git clone git@github.com:confluentinc/cli-release.git $(CLI_RELEASE) && \
 	go run $(CLI_RELEASE)/cmd/releasenotes/formatter/main.go $(CLI_RELEASE)/release-notes/$(VERSION_NO_V).json github > $(DIR)/release-notes.txt && \
 	GORELEASER_KEY=$(GORELEASER_KEY) S3FOLDER=$(S3_STAG_FOLDER_NAME)/confluent-cli GITHUB_TOKEN=$(token) DRY_RUN=$(DRY_RUN) goreleaser release --clean --release-notes $(DIR)/release-notes.txt --timeout 60m && \
-	$(call dry-run,gh release upload $(VERSION) prebuilt/*.deb prebuilt/*.rpm)
+	cd prebuilt && \
+	sha256sum confluent-cli_$(VERSION_NO_V)*.deb confluent-cli-$(VERSION_NO_V)*.rpm >> ../dist/confluent_$(VERSION_NO_V)_checksums.txt && \
+	cd - && \
```

Comment:
```suggestion
	sha256sum prebuilt/confluent-cli_$(VERSION_NO_V)*.deb prebuilt/confluent-cli-$(VERSION_NO_V)*.rpm >> dist/confluent_$(VERSION_NO_V)_checksums.txt && \
```

Reply:
This causes `prebuilt/` to show up in the checksums file because `sha256sum` prints the file path instead of just the file name.

Reply:
Alright I changed it to this but I also added a `sed` command to remove `prebuilt/`.